arrow-docs/docs/fx/README.md

```diff
@@ -87,158 +92,75 @@ suspend fun greet(): Unit {
 }
 ```
 
-#### `fx` composition
-
-Side effects can be composed and turned into pure values in `fx` blocks.
-
-#### Turning side effects into pure values with `effect`
-
-`effect` wraps the effect and turns it into a pure value by lifting any `suspend () -> A` user-declared side effect into an `IO<A>` value.
-
-```kotlin:ank:playground
-import arrow.fx.IO
-import arrow.fx.extensions.fx
-//sampleStart
-suspend fun sayHello(): Unit =
-  println("Hello World")
-
-suspend fun sayGoodBye(): Unit =
-  println("Good bye World!")
-
-fun greet(): IO<Nothing, Unit> =
-  IO.fx<Unit> {
-    val pureHello = effect { sayHello() }
-    val pureGoodBye = effect { sayGoodBye() }
-  }
-//sampleEnd
-fun main() {
-  println(greet())
-}
-```
+## Executing effectful programs
 
-When we capture suspended side effects as `IO` values with `effect`, we can pass them around and compose them until we are ready to apply the effects. At this point, nothing has happened because `greet` and `effect` are lazy values.
+The `greet` program is ready to run as soon as the user is ready to commit to an execution strategy that is either `blocking` or `non-blocking`.
+`blocking` execution strategies will block the current thread that's waiting for the program to yield a value, whereas `non-blocking` strategies will immediately return and perform the program's work without blocking the current thread.
 
-#### Applying side effects with `!effect`
+Since both blocking and non-blocking execution scenarios perform side effects, we consider running effects as an `unsafe` operation.
```

Comment:
Is `unsafe` here really code or not anymore given the new `Environment` api? if not I wouldn't use backticks.

Reply:
Yes, you're right it should be:

",we consider running effects with the `unsafeXX` operators unsafe", or I should remove the backticks. I think removing the backticks would make it more clear here, no?

Reply:
No backticks makes It more clear